[BE] Get list of job applicants

nspc97 opened the pull request:
[BE] Get list of job applicants
https://trello.com/c/ycA4ZSKm/51-be-get-list-of-job-applicants

Commit:
[BE] Apply for a job
https://trello.com/c/E1dTAfCz/50-be-apply-for-a-job

Commit:
[BE] Apply for a job
https://trello.com/c/E1dTAfCz/50-be-apply-for-a-job

Documentation annotations fixed.

Commit:
[BE] Get list of job applicants
https://trello.com/c/ycA4ZSKm/51-be-get-list-of-job-applicants

SpartakusMd commented on src/Controller/ActivityController.php:
```diff
@@ -479,4 +486,25 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
         return new JsonResponse(['message' => 'Applied with success!'], Response::HTTP_OK);
     }
+
+    /**
```
Missing `@SWG\Schema`

SpartakusMd commented on src/Controller/ActivityController.php:
```diff
@@ -479,4 +486,25 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
+    public function listOfApplicants(Activity $activity, ActivityUserRepository $activityUserRepo): JsonResponse
+    {
+        $applicants = $activityUserRepo->getApplicantsForActivity($activity);
+        $context = SerializationContext::create();
```
You are sure you don't need a context? What info is returned about the applicants?

Commit:
[BE] Get list of job applicants
https://trello.com/c/ycA4ZSKm/51-be-get-list-of-job-applicants

SpartakusMd commented on src/Controller/ActivityController.php:
```diff
@@ -488,16 +482,49 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
+     *     description="Successfull operation!",
+     *     @SWG\Schema(
+     *     type="array",
+     *     @Model(type=User::class, groups={"UserDetail"}),
```
`UserList` should be used as there's no need to show all the details about the user. Make sure it exists.

SpartakusMd commented on src/Controller/ActivityController.php:
```diff
@@ -488,16 +482,49 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
-        $context = SerializationContext::create();
+
+        /** @var SerializationContext $context */
+        $context = SerializationContext::create()->setGroups(array('UserDetail'));
```
UserDetail -> UserList

Commit:
[BE] Get list of job applicants
https://trello.com/c/ycA4ZSKm/51-be-get-list-of-job-applicants

Add group UserList

SpartakusMd commented:
Add the alias UserList in https://github.com/p5chi-auf/PentaBee-api/blob/develop/config/packages/nelmio_api_doc.yaml

Commit:
[BE] Get list of job applicants
https://trello.com/c/ycA4ZSKm/51-be-get-list-of-job-applicants

Add model UserList in documentation
